build.gradle

```diff
@@ -76,10 +74,27 @@ dependencies {
 //    implementation fg.deobf("_:Patchouli:1.18.2-71.1")
 //    implementation fg.deobf("_:TerraFirmaCraft:Forge-1.18.2-2.1.11-beta")
 //    implementation fg.deobf("_:Cartography:1.18.2-0.4.0-alpha3")
+
+
+    // Apply Mixin AP
+    annotationProcessor 'org.spongepowered:mixin:0.8.5:processor'
```

Comment:
Need this to generate the refmap, which is in turn needed to the mixins know wtf they're looking at when applying their changes in the obfuscated environment

Reply:
Oh so that's why..... great.

Reply:
That wasn't the only change needed to make the refmap generate, but it's definitely a key one!

My understanding is that the other Minecraft classes we were injecting into were still working without the refmap because they were being injected into constructors, which means their `@Inject(method` was already being defined partially obfuscated because they're a special case, and the Rubidium class didn't require obfuscation at all. That's why only `ChunkHolderMixin` and `ServerLevelMixin` were affected, not the other mixins. Also, pretty much only Injectors require a refmap for Mixin, since they define their target in a string and so can't be evaluated at compile time.